packages/babel-preset-razzle/dependencies.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict';
+
+module.exports = function(api, opts) {
```

Comment:
I think we ought to just ship with babel-preset-react-app going forward. Some of the issues I had early on with razzle appear to be resolved (HMR would break). We should probs just export it for continuity and backwards compat. No need to copy and paste I think.

Reply:
Yes, that would be a much better alternative to also reduce the maintenance